### DISCUSSION TITLE
Communication between servers

### DISCUSSION THREAD

**Discussion by Pingear** [serverpod/serverpod#562] [Q&A] [answered]
Hello)
Is the possibility of communication between servers implemented, or is it planned?

**Comment by yahu1031** [comment]
Like communicating 2 servers with each other?
> PS: Please create a discussion next time if it is a question.

**Reply by Pingear** [reply]
Oh, I forgot about the discussion, you're right)

**Comment by vlidholt** [comment, marked as answer]
Yes! This is easiest done with the Redis integration which provides messaging between servers through Serverpod's message central.

You can read more here: https://docs.serverpod.dev/concepts/streams#internal-server-messaging

**Reply by Pingear** [reply]
Thank You)
